jenkins.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
No need for shebang. This file must be sourced and having the shebang would suggest otherwise.